Assets/Scripts/Isamu/Utils/MonoBehaviourExtensions.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections;
+using UnityEngine;
+
+public static class MonoBehaviourExtensions
```

Comment:
This is just a static helper class that contains various coroutine wrapper functions. These can be called from `MonoBehaviour` classes to use some simple coroutine functionality without necessarily writing new coroutines from scratch every time.